cmd/kubeless/deploy.go

```diff
@@ -85,6 +100,9 @@ func init() {
 	deployCmd.Flags().StringP("runtime", "", "", "Specify runtime")
 	deployCmd.Flags().StringP("handler", "", "", "Specify handler")
 	deployCmd.Flags().StringP("from-file", "", "", "Specify code file")
+	deployCmd.Flags().StringP("description", "", "", "Specify description of the function")
+	deployCmd.Flags().StringSliceP("label", "", []string{}, "Specify labels of the function")
```

Comment:
`label` and `env` help strings need an example of the expected input syntax, since it is no longer just a simple scalar value.